assets/js/index.js

```diff
@@ -0,0 +1,914 @@
+			var menuModal = document.querySelector('.menu-modal');
+			var headerToggles = document.querySelector('.header-toggles');
+			var searchModal = document.querySelector('.search-modal');
+			if (menuModal.classList.contains('.active')) {
+				if (!menuModal.contains(element) && !headerToggles.contains(element)) {
+					document.querySelector('.nav-toggle').focus();
+				}
+			} else if (!searchModal.classList.contains('.active')) {
+				if (!searchModal.contains(element)) {
+					searchModal.querySelector('.search-field').focus();
+				}
+			}
+		});
+	},
+
+}; // twentytwenty.focusManagement
+
+/*	-----------------------------------------------------------------------------------------------
+	Intrinsic Ratio Embeds
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.intrinsicRatioVideos = {
+
+	init: function () {
+		this.makeFit();
+
+		window.addEventListener('fit-videos', function () {
+			this.makeFit();
+		}.bind(this));
+
+		window.addEventListener('resize', function () {
+			this.makeFit();
+		}.bind(this));
+	},
+
+	makeFit: function () {
+		document.querySelectorAll('iframe, object, video').forEach(function (video) {
+			var container = video.parentNode;
+
+			// Skip videos we want to ignore
+			if (video.classList.contains('intrinsic-ignore') || video.parentNode.classList.contains('intrinsic-ignore')) {
+				return true;
+			}
+
+			if (!video.dataset.origwidth) {
+				// Get the video element proportions
+				video.setAttribute('data-origwidth', video.width);
+				video.setAttribute('data-origheight', video.height);
+			}
+
+			var iTargetWidth = container.offsetWidth;
+
+			// Get ratio from proportions
+			var ratio = iTargetWidth / video.dataset.origwidth;
+
+			// Scale based on ratio, thus retaining proportions
+			video.style.width = iTargetWidth + 'px';
+			video.style.height = video.dataset.origheight * ratio + 'px';
+		});
+	},
+
+}; // twentytwenty.instrinsicRatioVideos
+
+/*	-----------------------------------------------------------------------------------------------
+	Smooth Scroll
+--------------------------------------------------------------------------------------------------- */
+
+twentytwenty.smoothScroll = {
+
+	init: function () {
+		// Scroll to anchor
+		this.scrollToAnchor();
+
+		// Scroll to element
+		this.scrollToElement();
+	},
+
+	// Scroll to anchor
+	scrollToAnchor: function () {
+		var anchorElements = document.querySelectorAll('a[href*="#"]');
+		var anchorElementsList = Array.prototype.slice.call(anchorElements);
+		anchorElementsList.filter(function (element) {
+			if (element.href === '#' || element.href === '#0' || element.classList.contains('.do-not-scroll') || element.classList.contains('skip-link')) {
+				return false;
+			}
+			return true;
+		}).forEach(function (element) {
+			element.addEventListener('click', function (event) {
+				// On-page links
+				if (window.location.hostname === event.target.hostname) {
```

Comment:
The `event.target.hostname` here is `undefined` because the target is the `span.to-the-top-long` tag. The event should be on the `a` tag, it would work as intended if you add `pointer-events: none` on both span tags wrapped by this element.